public/styles/message-box.css

```
@@ -0,0 +1,17 @@
+.message-box {
+  display: block;
+  width: 100%;
+  text-align: center;
+  padding-top: 56px;
+}
+
+.message-box span {
+  margin-top: 16px;
+  display: block;
+  font-family: Poppins;
```

Stylelint (17.5.0)

[error] Unexpected missing generic font family (font-family-no-missing-generic-family-keyword)

(font-family-no-missing-generic-family-keyword)